canister/src/heartbeat.rs

```diff
@@ -228,6 +232,30 @@ fn maybe_get_successors_request() -> Option<GetSuccessorsRequest> {
     })
 }
 
+#[cfg(target_arch = "wasm32")]
```

Comment:
We encapsulate all the system api calls in `runtime.rs`, so I'd move this logic there.

Reply:
Done.